src/config-site/windows.cmake

```diff
@@ -135,7 +135,7 @@ VISIT_OPTION_DEFAULT(VISIT_HDF4_DIR     ${VISITHOME}/hdf/4.2.13)
 VISIT_OPTION_DEFAULT(VISIT_HDF4_LIBDEP  
       JPEG_LIBRARY_DIR JPEG_LIB
       SZIP_LIBRARY_DIR SZIP_LIB
-      ZLIB_LIBRARY_DIR ZLIB_LIBRARY
+      ZLIB_LIBRARY_DIR ZLIB_LIB
```

Comment:
Am confused...in preceding CMakeLists.txt files `ZLIB_LIB` was replaced with `ZLIB_LIBRARY`. Here, the reverse appears to be happening.

Reply:
our dependency checking requires listing of the library location, then the library, separately.  When adding libraries to target_link_libraries, cmake prefers full path to library.  ZLIB_LIB is just the library name, ZLIB_LIBRARY is the full path.